bot.js

```diff
@@ -62,9 +62,12 @@ client.on('message', message => {
 
 	if (message.content === `${prefix}groups`) {
 		let [differenceMessage, chaosPinkBeanGroup, groupMessage] = [formatDifferenceMessage(computeDifference(groups[0], groups[1], true)), assemblePinkBeanGroup([...leaders, ...pool]), ""]
```

Comment:
suggested implementation for computeDifference if you want to display stuff:
computeDifference(groups[0], groups[1], true) => computeDifference()
```
function computeDifference(){
        const totalRanks = groups.map(group => totalRank(group))
	const lowestRank = Math.min(...totalRanks)
	const highestRank = Math.max(...totalRanks)
	return highestRank - lowestRank
}
```